nrf_modem/Kconfig

```diff
@@ -33,6 +33,10 @@ config NRF_MODEM_LINK_BINARY_DECT_PHY
 	select EXPERIMENTAL
 	bool "DECT PHY"
 
+config NRF_MODEM_LINK_BINARY_DECT_NRP
```

Comment:
To be aligned with agreed naming, I'd prpose this to be renamed as:
NRF_MODEM_LINK_BINARY_DECT